core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1448,6 +1448,16 @@ public Map<String, Map<String, Long>> getTopicRequestsCounts(
       updateMap(operationTypeCountsMap, topicRequestsOperationTypObj);
 
       operationTypeCountsMap.put(RequestOperationType.CLAIM.value, assignedToClaimReqs);
+      if (RequestMode.MY_APPROVALS == requestMode) {
+        // Make sure to remove any requests which the requestor can not approve
+        List<Object[]> connectorApprovalStatusObj =
+            topicRequestsRepo.findOtherRequestorsTopicRequestsGroupByStatus(
```

Comment:
why not to replace with count query?
Count query is much faster since it doesn't need to load all the field values

Reply:
I changed these to count only, a null value tells us that the count is actually == 0